ElectronicObserver/Window/Tools/DialogAlbumMasterEquipment/EquipmentUpgrade/AlbumMasterEquipmentUpgradeTranslationViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using ElectronicObserver.ViewModels.Translations;
+
+namespace ElectronicObserver.Window.Tools.DialogAlbumMasterEquipment.EquipmentUpgrade
+{
+	public class AlbumMasterEquipmentUpgradeTranslationViewModel : TranslationBaseViewModel
+    {
+		public string ZeroToFive => "0 \uff5e 5";
```

Comment:
I'm guessing this is either sonar or R# complaining about →. We should probably remove that rule, I mark it as won't fix whenever it comes up.

Reply:
It actually got replaced automatically (probably by R# indeed)